src/views/Graph.vue

```diff
@@ -1335,92 +1326,89 @@ export default {
         // ...loop through collapsed families...
         for (const family of this.collapseFamily) {
           // ...get the node from the index...
-          const indexSearch = Object.values(this.cylcTree.$index).find((node) => {
-            return node.name === family && node.tokens.cycle === cycle
-          })
+          const indexSearch = Object.values(this.cylcTree.$index).find(
+            (node) => node.name === family && node.tokens.cycle === cycle
+          )
+          if (!indexSearch) continue
```

Comment:
Does linting really allow omitting the braces?

Reply:
Yes for control statements on the same line as the condition